---a/net/minecraft/item/ItemSaddle.java
+++b/net/minecraft/item/ItemSaddle.java
@@ -5,41 +5,34 @@
 import net.minecraft.entity.passive.EntityPig;
 import net.minecraft.entity.player.EntityPlayer;
 
-public class ItemSaddle extends Item
-{
+public class ItemSaddle extends Item {
+
     private static final String __OBFID = "CL_00000059";
 
-    public ItemSaddle()
-    {
+    public ItemSaddle() {
         this.maxStackSize = 1;
         this.setCreativeTab(CreativeTabs.tabTransport);
     }
 
-    public boolean itemInteractionForEntity(ItemStack stack, EntityPlayer playerIn, EntityLivingBase target)
-    {
-        if (target instanceof EntityPig)
-        {
-            EntityPig entitypig = (EntityPig)target;
+    public boolean itemInteractionForEntity(ItemStack stack, EntityPlayer playerIn, EntityLivingBase target) {
+        if (target instanceof EntityPig) {
+            EntityPig entitypig = (EntityPig) target;
 
-            if (!entitypig.getSaddled() && !entitypig.isChild())
-            {
+            if (!entitypig.getSaddled() && !entitypig.isChild()) {
                 entitypig.setSaddled(true);
                 entitypig.worldObj.playSoundAtEntity(entitypig, "mob.horse.leather", 0.5F, 1.0F);
                 --stack.stackSize;
             }
 
             return true;
-        }
-        else
-        {
+        } else {
             return false;
         }
     }
 
-    public boolean hitEntity(ItemStack stack, EntityLivingBase target, EntityLivingBase attacker)
-    {
-        this.itemInteractionForEntity(stack, (EntityPlayer)null, target);
+    public boolean hitEntity(ItemStack stack, EntityLivingBase target, EntityLivingBase attacker) {
+        this.itemInteractionForEntity(stack, (EntityPlayer) null, target);
         return true;
     }
-}
+}
